http/handler.go

```diff
@@ -664,26 +664,28 @@ func respondStandby(core *vault.Core, w http.ResponseWriter, reqURL *url.URL) {
 
 // getTokenFromReq parse headers of the incoming request to extract token if present
 // it accepts Authorization Bearer (RFC6750) and X-Vault-Token header
-func getTokenFromReq(r *http.Request) (string, error) {
+func getTokenFromReq(r *http.Request) (string, bool, error) {
```

Comment:
This `error` response is always nil and could be removed.

Reply:
Done.